Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/README.md

```diff
@@ -5843,6 +5844,77 @@ Alert.ReadWrite.All
 
+### microsoft-atp-offboard-machine
+---
+Offboard a machine from microsoft ATP.
+
+##### Required Permissions
+Machine.Offboard	
+
+##### Base Command
+
+`microsoft-atp-offboard-machine`
+##### Input
+
+| **Argument Name** | **Description**                                                                                                                                            | **Required** |
+| --- |------------------------------------------------------------------------------------------------------------------------------------------------------------| --- |
+| machine_id | A comma-separated list of machine IDs to be used for offboarding. e.g., 0a3250e0693a109f1affc9217be9459028aa8426,0a3250e0693a109f1affc9217be9459028aa8424. | Required | 
+| comment | A comment to associate with the action.                                                                                                                    | Required |
+
+
+##### Context Output
+
+| **Path** | **Type** | **Description**                                            |
+| --- | --- |------------------------------------------------------------|
+| MicrosoftATP.MachineAction.ID | String | The machine action ID.                                     | 
+| MicrosoftATP.MachineAction.Type | String | Type of the machine action.                                | 
+| MicrosoftATP.MachineAction.Scope | Unknown | Scope of the action.                                       | 
+| MicrosoftATP.MachineAction.Requestor | String | The ID of the user that executed the action.               | 
+| MicrosoftATP.MachineAction.RequestorComment | String | Comment that was written when issuing the action.          | 
+| MicrosoftATP.MachineAction.Status | String | The current status of the command.                         | 
+| MicrosoftATP.MachineAction.MachineID | String | The machine ID on which the action was executed.           | 
+| MicrosoftATP.MachineAction.ComputerDNSName | String | The machine DNS name on which the action was executed.     | 
+| MicrosoftATP.MachineAction.CreationDateTimeUtc | Date | The date and time when the action was created.             | 
+| MicrosoftATP.MachineAction.LastUpdateTimeUtc | Date | The last date and time when the action status was updated. | 
+| MicrosoftATP.MachineAction.cancellationDateTimeUtc | Date | The date and time when the action was canceled.      |
+| MicrosoftATP.MachineAction.RelatedFileInfo | String | The file info.                                             | 
+| MicrosoftATP.MachineAction.troubleshootInfo | String | Troubleshooting information.                               |
```

Comment:
After you change the keys of the outputs in the `yml` file, please also change the `REDME` file